AGENTS.md

```diff
@@ -2,6 +2,34 @@
 
 - To test opencode in the `packages/opencode` directory you can run `bun dev`
 
+## Code Formatting
+
+- After changing TypeScript code, run `bun run prettier --ignore-unknown --write .` to format the code
+- This ensures consistent formatting across the codebase
```

Comment:
There's a script to format the code in GH actions, the documentation should replicate the way to run it instead of proposing custom formatting tool.